api/autoname.go

```diff
@@ -847,6 +857,29 @@ func nameByTagUID(exec boil.Executor, uid string, cNumber *int) (map[string]stri
 	return names, nil
 }
 
+func nameByLikutimUID(exec boil.Executor, uid string) (map[string]string, error) {
```

Comment:
Nothing here is special to type likutim. 
Maybe rename to something like `nameByUnit` so it sounds more generic